powerquery-docs/connectors/postgresql.md

```diff
@@ -23,11 +23,11 @@ ms.author: dougklo
 
```

Comment:
Should you add yourself in the author?

Reply:
I'm already in the metadata as the author and ms.author.